src/quacc/recipes/common/phonons.py

```diff
@@ -79,17 +84,23 @@ def phonon_subflow(
     PhononSchema
         Dictionary of results from [quacc.schemas.phonons.summarize_phonopy][]
     """
-
-    phonon = get_phonopy(
+    phonon, atoms_to_add = get_phonopy(
         atoms,
+        fixed_atoms=fixed_atoms,
         min_lengths=min_lengths,
         supercell_matrix=supercell_matrix,
         symprec=symprec,
         displacement=displacement,
         phonopy_kwargs=phonopy_kwargs,
     )
+
+    fixed_atoms = np.full(len(phonon.supercell), False)
+    fixed_atoms = np.append(fixed_atoms, [True] * len(atoms_to_add))
+    fixed_atoms = fixed_atoms.astype(bool)
+
     supercells = [
-        phonopy_atoms_to_ase_atoms(s) for s in phonon.supercells_with_displacements
+        phonopy_atoms_to_ase_atoms(s) + atoms_to_add
+        for s in phonon.supercells_with_displacements
     ]
```

Comment:
I think I need to return to this later to fully digest what's going on, but I'll trust you on it for the time being.